.gitignore

```diff
@@ -18,6 +18,10 @@ dist/
 build/
 lib/
 coverage/
+target
+test/rust/
+metaplex-tests-ledger
```

Comment:
Do we need this? Amman writes the ledger to a `/tmp` dir so this folder shouldn't be created.